data/FilterListTag.json

```diff
@@ -10086,5 +10086,185 @@
   {
     "filterListId": 2036,
     "tagId": 6
+  },
+  {
+    "filterListId": 2047,
+    "tagId": 14
+  },
+  {
+    "filterListId": 2048,
+    "tagId": 6
+  },
+  {
+    "filterListId": 2049,
+    "tagId": 6
+  },
+    {
+    "filterListId": 2050,
+    "tagId": 14
+  },
+  {
+    "filterListId": 2051,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2052,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2053,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2054,
+    "tagId": 3
+  },
+    {
+    "filterListId": 2055,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2056,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2057,
+    "tagId": 3
+  },
+  {
+    "filterListId": 2058,
+    "tagId": 24
+  },
+  {
+    "filterListId": 2059,
+    "tagId": 24
+  },
+    {
+    "filterListId": 205,
+    "tagId": 
```

Comment:
The build error [does give a line number](https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=2976&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=27) if you know what you are looking for. I'll try to figure out a way to make that easier to find.

But, regardless, I think this lack of tagId is an issue now.

Reply:
I noticed the line number, but didn't know which file to look for it in.

I'll try to fix it now.

Reply:
Yeah, the error messages could definitely be improved...

Reply:
Okay, I think I saw the real error now. This sure became a bit strange. Give me another minute.